frontend/packages/ux-editor/src/components/config/editModal/EditComponentId.tsx

```diff
@@ -49,11 +63,32 @@ export const EditComponentId = ({
         }
       }}
       renderField={({ fieldProps }) => (
-        <Textfield
-          {...fieldProps}
-          name={`component-id-input${component.id}`}
-          onChange={(e) => fieldProps.onChange(e.target.value, e)}
-        />
+        <div ref={wrapperRef}>
+          {!isEditMode ? (
+            <StudioButton
```

Comment:
I wonder if the StudioButton should be placed outside the `FormField` 🤔 So that if editMode is `true`, return the original `FormField`, and if editMode is `false` we return this other view. Semantically at least, I would not define this as a form field until it is actually editable. @framitdavid do you have any thoughts on this? How have we done this in other cases?

Reply:
The reason I believe we can avoid using the `document` API and `eventListeners` now is because we can use onBlur on IconTextfieldProps instead. 😊

Reply:
Makes sense! Would you then wrap the `TextfieldToggleView` component in the `FormField` component when implementing in `ux-editor` to get the validations etc. for the field?

Reply:
I believe we can actually remove the entire `FormField` and instead perform validation directly in `EditComponentId`. This way, we can send the error directly to `<TextfieldToggleView/>`, and the text field from the `digdir/design system` will render the error message with the correct styling. 😊

Reply:
@framitdavid , It sounds good 😊, I will try to implement your suggestions

Reply:
I agree with @framitdavid that it might be smart to move it to ```@studio/components``` 😄 
I also belive we should use the name ```StudioTextfieldToggleView``` instead so we follow the correct naming convention 😄